spec/test_like.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+RSpec.describe Like do
+  context 'Test class Like: ' do
+    before :each do
+      @like = Like.new(user_id: 1, post_id: 2)
+    end
+
+    it 'test attrs' do
+      expect(@like.user_id).to eql(1)
+      expect(@like.post_id).to eql(2)
+    end
+  end
+end
```

Comment:
Please fix the error these 2 your files to avoid loading error
- `test_lik.rb` and `test_coment.rb`

![image](https://github.com/khanh-devos/rail-blog-app/assets/78251927/11432fea-8e77-40eb-a5b0-af779524650f)

You can fix that by adding [rails_helper ](https://medium.com/@brett_codes/rspecs-spec-helper-vs-rails-helper-explained-2337282d14ce)
```ruby
require 'rails_helper'

RSpec.describe Comment do
  before :each do
    @text = 'text'
    @comment = Comment.new(text: @text)
  end

  it 'test attrs' do
    expect(@comment.text).to eql(@text)
  end
end
```
```

Reply:
This is very great idea @abdulhamiid